components/bt/host/bluedroid/btc/profile/std/sdp/btc_sdp.c

```diff
@@ -248,7 +286,7 @@ static int free_sdp_slot(int id)
         handle = -1;
     }
     osi_free(slot);
-    slot = NULL;
+    sdp_local_param.sdp_slots[id] = NULL;
```

Comment:
A mutex is required.

Reply:
Hmm, under this line I moved the mutex release from several lines above.
Seems to me that it doesn't break anything.